### DISCUSSION TITLE
Можно ли использовать как полноценный Прокси сервер?

### DISCUSSION THREAD

**Discussion by CloudGuver** [hufrea/byedpi#69] [Q&A]
Приветствую, в вашем приложении есть возможнось создания прокси сервера, вопрос могу ли Я его использовать как полноценный прокси, что бы моё приложение гнало весь трафик через ваше приложение если оно запущено?

**Comment by zetcamp** [comment]
Так прямо в описании:

The program is a local SOCKS proxy server.

**Comment by lufog** [comment]
@CloudGuver если имеется в виду, запустить программу, и она автоматически перенаправит весь трафик через себя как например GoodbyeDPI, то нет. Это именно что локальный прокси, который нужно явно указать в настройках системы (ну или браузера). Причем даже если указать прокси в настройках системы, это не гарантирует что все программы будут его использовать. Например, на Windows мне так и не удалось заставить Windows Store и UWP приложения использовать прокси. Но обычные приложения в большинстве случаев без проблем работают через прокси.

**Reply by jura1243** [reply]
> Например, на Windows мне так и не удалось заставить Windows Store и UWP приложения использовать прокси. Но обычные приложения в большинстве случаев без проблем работают через прокси.

у меня Windows Store и UWP приложения работают нормально через byedpi .

**Comment by krlvm** [comment]
> Например, на Windows мне так и не удалось заставить Windows Store и UWP приложения использовать прокси. 

https://github.com/krlvm/PowerTunnel/issues/102#issuecomment-1244411866

**Comment by ALV-036** [comment]
Keenetic  может отправить весь ваш трафик на локальный домашний Proxy server byedpi.
 
По инструкции настраиваете в нём прокси клиента и в политиках  перетаскиваете домашнее устройства на подключение с proxy, и всё.

https://help.keenetic.com/hc/ru/articles/7474374790300-%D0%9A%D0%BB%D0%B8%D0%B5%D0%BD%D1%82-%D0%BF%D1%80%D0%BE%D0%BA%D1%81%D0%B8

**Comment by ALV-036** [comment]
На  29_09_2024       byedpi  установленный на Ubuntu server    с параметрами  
 ./ciadpi-x86_64 --drop-sack --tfo --md5sig --no-domain --split 2 --split 3 --tlsrec 1+s --auto=torst --timeout 1

Youtube летает просто сказка.  Провайдер Воронеж-Ростелеком

    byedpi на Ubuntu является прокси сервером, с ПК  можно ссылаться на него через настройки Firefox.

На Android устройства можно поставить прокси клиент или внаглую загнать трафик от них на прокси сервер средствами модема Keenetic    
https://help.keenetic.com/hc/ru/articles/7474374790300-%D0%9A%D0%BB%D0%B8%D0%B5%D0%BD%D1%82-%D0%BF%D1%80%D0%BE%D0%BA%D1%81%D0%B8
